test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -82,7 +86,7 @@ public void before() {
         when(consent.getSignedOn()).thenReturn(ENROLLMENT.getMillis()); 
         
         userConsentDao = mock(UserConsentDao.class);
-        when(userConsentDao.getActiveUserConsent(any(String.class), any(StudyIdentifier.class))).thenReturn(consent);
+        when(userConsentDao.getActiveUserConsent(eq(HEALTH_CODE), any(SubpopulationGuid.class))).thenReturn(consent);
```

Comment:
In adding two different subpopulations, this was converted to look for specific arguments.